tests/e2e/objects/views/viewItems/KafkaClusterItem.ts

```diff
@@ -0,0 +1,4 @@
+import { ViewItem } from "./ViewItem";
+
+export class KafkaClusterItem extends ViewItem {
```

Comment:
Minor: unless we're adding some convenience locators/methods here, we could skip creating a dedicated `KafkaClusterItem` and use `new ViewItem(page, <cluster locator>)` in the new test. At the same time, it might not be a bad idea to have some of these actions set up:

<img width="505" height="395" alt="Image" src="https://github.com/user-attachments/assets/bebd3408-1436-4f62-a1dc-ce30357df81f" />

What are your thoughts there?